src/main/webapp/WEB-INF/datastore-indexes.xml

```diff
@@ -1,4 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+  - When adding a new index definition to support a new query, follow these steps:
+  - 1. Set autoGenerate to true.
+  - 2. Restart the dev server and run the new query.
+  -    The required index definition will be generated and saved in the file
+  -    "appengine-generated/datastore-indexes-auto.xml".
+  - 2. Copy the index definition from that file into this one.
```

Comment:
Based on our discussion, no need to add `(and tweak it if required)` as tweaking is only required under very special circumstances (complex queries that result in exploding indexes). Adding it in might confuse developers who are only developing simple queries, in which case the auto generated index definition is good enough.